app/views/home/_tos.html.erb

```diff
@@ -425,52 +425,41 @@
+      <p>Control over individual fanworks contained within a collection rests with their creators. If the verified creator of any individual fanwork contained within a collection requests its removal or alteration, the OTW will always comply with such a request in a reasonable period of time. We will also provide mechanisms allowing creators to claim their fanworks from such a collection and if desired to attach them to a new or existing Archive of Our Own account, edit them, orphan them, or delete them.</p>
     </li>
     <li>Parting from the OTW
-      <p>As noted in section e., control over individual fanworks contained within a collection always rests with their creators. This section applies to collections as a whole. If the collection's original maintainer decides that they no longer want to be affiliated with the OTW, or the OTW board decides they no longer wish to work with the original maintainer, the following procedures for dissolution will apply:</p>
+      <p>As noted in section 5, control over individual fanworks contained within a collection always rests with their creators. This section applies to collections as a whole. If the collection's original maintainer decides that they no longer want to be affiliated with the OTW, or the OTW board decides they no longer wish to work with the original maintainer, the following procedures for dissolution will apply:</p>
       <ol>
         <li>
-          <p>OTW will keep the content currently on the OTW server(s), whether in the Archive of Our Own or in a special collection.</p>
-        </li>
-        <li>
-          <p>OTW will keep the subdomain (e.g., foresmutters.transformativeworks.org) and subdirectory (e.g, opendoors.transformativeworks.org/foresmutters) URLs of special projects, pointing to the content currently on the OTW servers.</p>
-        </li>
-        <li>
-          <p>OTW will give the original maintainer back the original domain name, if any (e.g., www.foresmutters.org) or the archive or special project, as well as a copy of the current contents of the project (or all stories under the archive tag in the Archive of Our Own).</p>
-        </li>
+          <p> Open Doors will delete the imported archive’s collection on AO3. Any works imported to the AO3 already claimed, orphaned, or anonymized by AO3 users will remain in the Archive under control of the creators' accounts, or the orphan account. Open Doors will work with the original maintainer to delete any unclaimed works.        </li>
```

Comment:
There's an extra space after the <p> tag here, and it needs a </p> tag before the </li>